server/services/database/profile.js

```diff
@@ -22,7 +22,28 @@ const getAllProfiles = () => {
 	return pool.query("SELECT * FROM profile").then((result) => result.rows);
 };
 
+const getProfileById = (profileId) => {
+	return pool
+		.query("SELECT * FROM profile WHERE profile_public_id=$1", [profileId])
+		.then((result) => result.rows);
+};
+
+const getEducationById = (profileId) => {
+	return pool
+		.query("SELECT * FROM education WHERE profile_id=$1", [profileId])
+		.then((result) => result.rows);
+};
+
+const getWorkExperiencesById = (profileId) => {
```

Comment:
should be getWorkExperiencesByProfileId (to make it clearer that it is the profile_id that should be passed, not the id, and to make it clear it returns a array (because experience*s*)